src/states_screens/race_gui_base.cpp

```diff
@@ -1039,6 +1039,21 @@ void RaceGUIBase::drawPlayerIcon(AbstractKart *kart, int x, int y, int w)
                                                       NULL, true);
         }
     }
+
+	//lap flag for finished karts
+	if (kart->hasFinishedRace())
+	{
+        video::ITexture *icon_lapflag = irr_driver->getTexture(FileManager::GUI_ICON, "lap_flag.png");
+        if (icon_lapflag != NULL)
+		{
+            const core::rect<s32> rect(core::position2d<s32>(0, 0),
+                                       icon_lapflag->getSize());
+            const core::rect<s32> pos1(x - 20, y - 10, x + w - 20, y + w - 10);
+            draw2DImage(icon_lapflag,
+                                                      pos1, rect, NULL,
```

Comment:
weird indentation

Reply:
Thought so too, but it's the way these calls are indented in the function and I thought this is done intentionally.

Reply:
No, that's definitely not right, maybe it's a mistake or a leftover from a merge (merge tools are usually not indentation aware). It would be preferable to fix this in your code, even if there are issues elsewhere